src/app_engine/test_utilities.py

```diff
@@ -42,13 +42,19 @@ def tearDown(self):
     self.testbed.deactivate()
     analytics.report_event = self.oldReportEvent
 
-  def checkInvalidRequests(self, path, params):
+  def checkInvalidRequests(self, path, params, jsonResult=False):
```

Comment:
Which API returns this form of result? We probably should make it consistent with the other APIs.